src/components/Personality/PersonalityView.jsx

```diff
@@ -48,69 +131,41 @@ class PersonalityView extends Component {
 
     render() {
         const personality = this.state.personality;
+        const arraynovo = new Array().fill(100);
```

Comment:
I guess this is not supposed to be here.

Reply:
it isn't, thanks